docs/console/console-extensibility/locations.md

```diff
@@ -0,0 +1,69 @@
+---
+id: locations
+title: Extensible Locations
+sidebar_label: Extensible Locations
+---
+
+# Extensible Locations
+
+The Console allows to embed extensions at specific locations that already have menu items and groups that can also be used by our extensions. These locations are:
+
+- **Tenant** with locationId `tenant`
+- **Project** with locationId `project`
+- **Runtime** with locationId `runtime`
+
+## Tenant
+
+<div style={{display: 'flex', justifyContent: 'center'}}>
```

Comment:
these image about locations are very small and i should find a way to improve their presentation